presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestDistributedQueries.java

```diff
@@ -289,6 +289,12 @@ public void testExplainAnalyze()
         assertExplainAnalyze("EXPLAIN ANALYZE SHOW SESSION");
     }
 
+    @Test
+    public void testExplainAnalyzeVerbose()
```

Comment:
- Is there a point in executing this for many different connectors?

- Commits are out of order.